### ISSUE TITLE
ERROR: Error: Missing required parameters: location

### ISSUE THREAD

**Issue by gemlorenz10** [GoogleCloudPlatform/cloud-functions-emulator#190]
`$functions start ` - returns **ERROR: Error: Missing required parameters: location**
#### Stack Trace

`**ERROR:** Error: Missing required parameters: location
at Object.createAPIRequest (D:\path-to-project\functions\node_modules\googleapis\lib\apirequest.js:95:18)
    at Endpoint.<anonymous> (D:\path-to-project\functions\node_modules\googleapis\lib\discovery.js:64:29)
    at Promise (D:\path-to-project\functions\node_modules\@google-cloud\functions-emulator\src\client\rest-client.js:34:43)
    at getService.then (D:\path-to-project\functions\node_modules\@google-cloud\functions-emulator\src\client\rest-client.js:33:16)
    at process._tickCallback (internal/process/next_tick.js:109:7)`

#### Environment details
  - OS: **Windows 10**
  - Node.js version: **v6.11.5**
  - npm version: **5.6.0**
  - @google-cloud/functions-emulator version: **1.0.0-beta.2**

#### config.json
{
	"bindHost": "localhost",
	"host": "localhost",
	"idlePruneInterval": 60000,
	"logFile": "logs/cloud-functions-emulator.log",
	"maxIdle": 300000,
	"region": "us-central1",
	"restPort": 8008,
	"service": "rest",
	"storage": "configstore",
	"supervisorPort": 8010,
	"tail": false,
	"timeout": 20000,
	"useMocks": false,
	"verbose": false,
	"watch": true,
	"watchIgnore": [
		"node_modules"
	],
	"grpcPort": 8009,
	"projectId": "my-project-id"
}

#### defaults
{
  "bindHost": "localhost",
  "host": "localhost",
  "idlePruneInterval": 60000,
  "logFile": "logs/cloud-functions-emulator.log",
  "maxIdle": 300000,
  "region": "us-central1",
  "restPort": 8008,
  "service": "rest",
  "storage": "configstore",
  "supervisorPort": 8010,
  "tail": false,
  "timeout": 20000,
  "useMocks": false,
  "verbose": false,
  "watch": true,
  "watchIgnore": [
    "node_modules"
  ]
}

**Comment by gemlorenz10** [comment]
## [Solved]
Config files in `C:\Users\user name\\.config\configstore\\@google-cloud\functions-emulator` still points to an old firebase project.
## Steps I did

1. Clone the project in a separate computer.
2. Set ServiceKey
3. Run function-emulator
4. Copy all the config  files in `C:\Users\user name\.config\configstore\@google-cloud\functions-emulator`.
5. Paste them into the same path in your work computer.

Basically, I manually updated config files in my work computer. I don't know if there is an option to update them automatically/dynamically. If there is I hope you can show me how.
